IPFS-scholar.md

```diff
@@ -0,0 +1,61 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*IPFS-scholar/hfs-ipfs-scholar*
+
+## (1) Brief Project Description
+
+*Ipfs-scholar is a platform for academic publishing similar to SCI and EI. However, different from SCI, it avoids many tedious processes and enables researchers and students to quickly and freely publish their scientific research achievements. The system USES the cold and hot decentralized distributed storage mode of IPFS+Filecoin to break the status quo of data monopoly by a few giant enterprises, and decentralize the power, realizing the vision of data producer as data owner and beneficiary.*
+
+## (2) Link to Project UI
+
+*http://122.112.251.156:8022/#/.*
+
+## (3) What does your application/UI do?
+
+*Scholars can publish their research results through IPFs-Scholar, and they can also download some papers and data sets from websites and reward the authors.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*All the data we collect is open source and free, and all uploaded data are public data.*
```

Comment:
Can you specify the source of this data?

Reply:
We have a team that collects free and open datasets and other academic materials in astromedicine. And we have reached a preliminary agreement with a medical institution in the UK to store hospital data on our platform.